stable/activemq-artemis/templates/ci-ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    kubernetes.io/ingress.class: internal-ingress
+  name: {{ include "artemis.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - host: {{ include "artemis.fullname" . }}.ci.tarabutgateway.io
```

Comment:
other environments will also be included like, sandbox and prod and also sau. Better to interpolate the full host.

Reply:
Can we test the chart with this PR, or we should interpolate and then do the testing?

Reply:
@syed-faraz Can you check now?